proctor-common/src/main/java/com/indeed/proctor/common/model/PayloadExperimentConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.indeed.proctor.common.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+@Data
+@Builder
+public class PayloadExperimentConfig {
+    @Nullable private String priority;
```

Comment:
Is `priority` a String? Should it be an `integer`?

Reply:
Should be a string. The matrix stores the version as a string which will be used to populate. There was desire for also custom input and lexico sort for priority

Reply:
Just to confirm, we want lexico and **not** ordinal compare for `AppVersion` (for priority)?

Reply:
Hmmm, good catch right now should be ordinal but for the MVP phase 2 when we allow custom string we will want to allow lexico.